main.go

```diff
@@ -23,8 +23,17 @@ import (
 	"github.com/openshift/windows-machine-config-operator/pkg/metrics"
 	"github.com/openshift/windows-machine-config-operator/pkg/nodeconfig/payload"
 	"github.com/openshift/windows-machine-config-operator/version"
+	//+kubebuilder:scaffold:imports
 )
 
+// needed to run on the hostnetwork
+//+kubebuilder:rbac:groups=security.openshift.io,resources=securitycontextconstraints,resourceNames=hostnetwork,verbs=use
```

Comment:
I assume this resulted in a RBAC policy for `securitycontextconstraints`. How were we able to run on the host network without this?